example/src/test/java/gaffer/example/gettingstarted/analytic/LoadAndQuery1Test.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gaffer.example.gettingstarted.analytic;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import com.google.common.collect.Lists;
+import gaffer.commonutil.StreamUtil;
+import gaffer.data.element.Edge;
+import gaffer.exception.SerialisationException;
+import gaffer.graph.Graph;
+import gaffer.jsonserialisation.JSONSerialiser;
+import gaffer.operation.OperationException;
+import gaffer.operation.impl.add.AddElements;
+import gaffer.operation.impl.get.GetRelatedEdges;
+import org.hamcrest.core.IsCollectionContaining;
+import org.junit.Test;
+import java.util.List;
+
+public class LoadAndQuery1Test {
+    private static final String RESOURCE_PREFIX = "/example/gettingstarted/";
+    private static final String RESOURCE_EXAMPLE_PREFIX = RESOURCE_PREFIX + "1/";
+    private static final String GROUP = "data";
+    private static final String COUNT = "count";
+
+    @Test
+    public void shouldReturnExpectedEdges() throws OperationException {
+        // Given
+        final LoadAndQuery1 query = new LoadAndQuery1();
+
+        // When
+        final Iterable<Edge> results = query.run();
+
+        // Then
+        verifyResults(results);
+    }
+
+
+    @Test
+    public void shouldReturnExpectedEdgesViaJson() throws OperationException, SerialisationException {
+        // Given
+        final JSONSerialiser serialiser = new JSONSerialiser();
+        final AddElements addElements = serialiser.deserialise(StreamUtil.openStream(LoadAndQuery.class, RESOURCE_EXAMPLE_PREFIX + "json/load.json"), AddElements.class);
```

Comment:
Tests LoadAndQuery1-5 have duplicate shouldReturnExpectedEdgesViaJson methods. Maybe abstract this out. And create an @Before method to initialise the query to be used (currently in the //Given section).